### PR TITLE
update remaining maintenance docs

### PR DESCRIPTION
**Pull request by jdmooney**
update single-tier and multi-tier maintenance docs
based on cloudformation doc
fix links in single-tier setup

### CODE REVIEW

File: userguide/aws/how-to-maintain-a-multi-tier-system.adoc [hunk 1]
```diff
@@ -8,34 +8,21 @@ This guide explains how to maintain your multi-tier Kill Bill installation on AW
 
 To perform most maintenance tasks you will need to login to your EC2 instances. Be sure that SSH is enabled on Port 22 in your security group. Login procedures are described in your setup guide.
```
Comment (reshmabidikar) [inline userguide/aws/how-to-maintain-a-multi-tier-system.adoc:10]: > Login procedures are described in your setup guide.

Would be good to add a link to the corresponding setup guide.

File: userguide/aws/how-to-maintain-a-single-tier-system.adoc [hunk 2]
```diff
@@ -8,30 +8,6 @@ This guide explains how to maintain your single-tier Kill Bill installation on A
 
 To perform most maintenance tasks you will need to login to your EC2 instance. Be sure that SSH is enabled on Port 22 in your security group. Login procedures are described in your setup guide.
```
Comment (reshmabidikar) [inline userguide/aws/how-to-maintain-a-single-tier-system.adoc:10]: Would be good to add a link to the corresponding setup guide

File: userguide/aws/how-to-maintain-a-single-tier-system.adoc [hunk 3]
```diff
@@ -58,11 +34,19 @@ For each report there should be a line near the top with the following form:
...
 
-You can `start` or `stop` the services using similar commands, such as `sudo service kaui stop` to stop KAUI. If any of these services are not active, try starting it using the appropriate command.
+You can `start` or `stop` the services using similar commands, such as `sudo service kaui stop` to stop Kaui. If any of these services are not active, try starting it using the appropriate command.
 
 If all services are running, the next step is to collect several reports.
 
 
+=== Log Files
+
+The system maintains a series of logfiles that may be helpful when troubleshooting is needed. These logs may be forwarded to Kill Bill to help diagnose your problem.
```
Comment (reshmabidikar) [inline userguide/aws/how-to-maintain-a-single-tier-system.adoc]:
```suggestion
The system maintains a series of log files that may be helpful when troubleshooting is needed. These logs may be forwarded to Kill Bill to help diagnose your problem.
```

File: userguide/aws/how-to-maintain-a-single-tier-system.adoc [hunk 4]
```diff
@@ -58,11 +34,19 @@ For each report there should be a line near the top with the following form:
...
+
+Tomcat logs are under `/var/lib/tomcat/logs/`:
+
+* Kaui logs: `/var/lib/tomcat/logs/kaui.out`
+* Kill Bill server logs: `/var/lib/tomcat/logs/catalina.out`
```
Comment (reshmabidikar) [inline userguide/aws/how-to-maintain-a-single-tier-system.adoc]: Is this correct? Shouldn't the Kill Bill logs be `/var/lib/tomcat/logs/killbill.out`?

**Review by reshmabidikar — COMMENTED** [review]
Looks good, I've left a few comments. Also, the contents of `how-to-maintain-a-single-tier-system.adoc` and how-to-maintain-a-multi-tier-system.adoc` look the same to me. Do we need separate files?

**Review by reshmabidikar — COMMENTED** [review]
Looks good, I've left a few comments. Also, the contents of `how-to-maintain-a-single-tier-system.adoc` and how-to-maintain-a-multi-tier-system.adoc` look the same to me. Do we need separate files?

File: userguide/aws/how-to-maintain-a-multi-tier-system.adoc (hunk 1 above)
Comment (jdmooney) [reply userguide/aws/how-to-maintain-a-multi-tier-system.adoc:10]: I agree. I will add the link.

File: userguide/aws/how-to-maintain-a-single-tier-system.adoc (hunk 2 above)
Comment (jdmooney) [reply userguide/aws/how-to-maintain-a-single-tier-system.adoc:10]: I agree.

Comment (jdmooney) [reply userguide/aws/how-to-maintain-a-single-tier-system.adoc:10]: fixed

File: userguide/aws/how-to-maintain-a-multi-tier-system.adoc (hunk 1 above)
Comment (jdmooney) [reply userguide/aws/how-to-maintain-a-multi-tier-system.adoc:10]: fixed

File: userguide/aws/how-to-maintain-a-single-tier-system.adoc (hunk 3 above)
Comment (jdmooney) [reply userguide/aws/how-to-maintain-a-single-tier-system.adoc]: fixed

Comment (jdmooney) [reply userguide/aws/how-to-maintain-a-single-tier-system.adoc]: fixed

File: userguide/aws/how-to-maintain-a-single-tier-system.adoc (hunk 4 above)
Comment (jdmooney) [reply userguide/aws/how-to-maintain-a-single-tier-system.adoc]: Both logs exist. Catalina is the more comprehensive. I tried to clarify.

**Comment by jdmooney** [comment]
There are a few differences between the maintain docs for single-tier and multi-tier. The main difference is that the single-tier implementation includes the database and load balancer in the same instance. They could be combined, but there would be times when it would say "this applies to the xxx version only".

**Comment by reshmabidikar** [comment]
> There are a few differences between the maintain docs for single-tier and multi-tier. The main difference is that the single-tier implementation includes the database and load balancer in the same instance. They could be combined, but there would be times when it would say "this applies to the xxx version only".

Got it. Just that, if any updates are required to the docs in the future, both docs would need to be updated consitently. Another idea is to move the common content to a separate file and include it in both files via the `include`directive. See https://github.com/killbill/killbill-docs/blob/v3/userguide/subscription/userguide_subscription.adoc for an example.

File: userguide/aws/how-to-maintain-a-multi-tier-system.adoc [hunk 5]
```diff
@@ -6,36 +6,23 @@ This guide explains how to maintain your multi-tier Kill Bill installation on AW
 
 == Login
 
-To perform most maintenance tasks you will need to login to your EC2 instances. Be sure that SSH is enabled on Port 22 in your security group. Login procedures are described in your setup guide.
+To perform most maintenance tasks you will need to login to your EC2 instances. Be sure that SSH is enabled on Port 22 in your security group. Login procedures are described in your setup guide  (see https://docs.killbill.io/latest/how-to-set-up-a-multi-tier-system.html#_4_5_login_to_an_instance[4.5. Login to an Instance]).
```
Comment (reshmabidikar) [inline userguide/aws/how-to-maintain-a-multi-tier-system.adoc]:
```suggestion
To perform most maintenance tasks you will need to login to your EC2 instances. Be sure that SSH is enabled on Port 22 in your security group. Login procedures are described in your setup guide (See https://docs.killbill.io/latest/how-to-set-up-a-multi-tier-system.html#_4_5_login_to_an_instance[4.5. Login to an Instance]).
```

File: userguide/aws/how-to-maintain-a-single-tier-system.adoc [hunk 6]
```diff
@@ -6,32 +6,8 @@ This guide explains how to maintain your single-tier Kill Bill installation on A
 
 == Login
 
-To perform most maintenance tasks you will need to login to your EC2 instance. Be sure that SSH is enabled on Port 22 in your security group. Login procedures are described in your setup guide.
+To perform most maintenance tasks you will need to login to your EC2 instance. Be sure that SSH is enabled on Port 22 in your security group. Login procedures are described in your setup guide  (see https://docs.killbill.io/latest/how-to-set-up-a-single-tier-system.html#step_7_login_to_your_instance[Step 7: Login to an Instance])
```
Comment (reshmabidikar) [inline userguide/aws/how-to-maintain-a-single-tier-system.adoc]:
```suggestion
To perform most maintenance tasks you will need to login to your EC2 instance. Be sure that SSH is enabled on Port 22 in your security group. Login procedures are described in your setup guide (See https://docs.killbill.io/latest/how-to-set-up-a-single-tier-system.html#step_7_login_to_your_instance[Step 7: Login to an Instance])
```

**Comment by reshmabidikar** [comment]
> > There are a few differences between the maintain docs for single-tier and multi-tier. The main difference is that the single-tier implementation includes the database and load balancer in the same instance. They could be combined, but there would be times when it would say "this applies to the xxx version only".
> 
> Got it. Just that, if any updates are required to the docs in the future, both docs would need to be updated consitently. Another idea is to move the common content to a separate file and include it in both files via the `include`directive. See https://github.com/killbill/killbill-docs/blob/v3/userguide/subscription/userguide_subscription.adoc for an example.

I understand that this might be a big change, so let's keep it on hold for now and revisit later.

**Review by reshmabidikar — APPROVED** [review]
Left very minor comments, the PR can be merged once those are addressed.

File: userguide/aws/how-to-maintain-a-multi-tier-system.adoc (hunk 5 above)
Comment (jdmooney) [reply userguide/aws/how-to-maintain-a-multi-tier-system.adoc]: fixed

File: userguide/aws/how-to-maintain-a-single-tier-system.adoc (hunk 6 above)
Comment (jdmooney) [reply userguide/aws/how-to-maintain-a-single-tier-system.adoc]: fixed

**Pull request merged**